transforms/universal/filter/src/local_pipeline.py

```diff
@@ -0,0 +1,51 @@
+import os
+import sys
```

Comment:
Where does this come from?

Reply:
Oops, it is probably a result of refactoring and rebases. I removed it.